.github/workflows/pr-check.yaml

```
@@ -137,15 +137,21 @@ jobs:
           echo "Version of @podman-desktop/tests-playwright to be used: $version"
           jq --arg version "$version" '.devDependencies."@podman-desktop/tests-playwright" = $version' package.json > package.json_tmp && mv package.json_tmp package.json
 
-      - name: Execute pnpm in Bootc Extension
+      - name: build extension image
         working-directory: ./podman-desktop-extension-bootc
```

actionlint (1.7.9)

unexpected key "working-directory" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)